arkane/encorr/data.py

```diff
@@ -400,6 +416,29 @@
         'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
     },
 
+    # fitted using Colin's BAC algorithm and AC are included in the correction units kcal/mol
+    # BACs are fitted to neutral molecules with RMSE and MAE 5.50 and 3.58 kJ/mol respectively.
+    'dlpno-ccsd(t)/def2-tzvp': {
+    'Cl-F': -0.07683, 'N=N': 1.20853, 'N=O': 1.49303, 'Cl-H': 0.00108,
+    'Cl-N': -0.31383, 'Cl-O': 0.13871, 'Cl-S': -1.26284, 'C-O': -0.04381, 'C-N': -0.42627,
+    'C-Cl': -0.03151, 'Cl-Cl': -0.22919, 'C-H': 0.12906, 'C-F': 0.37444, 'C-C': -0.43463,
+    'S=S': -0.39416, 'C#O': 2.70341, 'C#N': 1.37471, 'C-S': -0.03749, 'C#C': -0.66008,
+    'O-O': 0.09663, 'C=S': 0.80888, 'H-S': 1.23648, 'C=O': 0.88066, 'C=N': 0.03998,
+    'H-N': -0.49564, 'H-O': -0.41183, 'H-H': 0.6263, 'N#N': 3.71325, 'N-N': 0.74915, 
+    'N-O': -0.62156, 'C=C': -0.63901, 'O=S': -1.39626, 'O-S': -1.37002, 'S-S': 0.1515,
+    'F-S': -0.68693, 'F-O': 0.09202, 'F-H': -1.68214, 'F-F': 0.95483, 'O=O': -2.64949
```

Comment:
please add a `,` at the end of the line (and also for the block below)

Reply:
These corrections are obtained from calculations that use `dlpno-ccsd(t)/def2-tzvp` for both energy calculations and geometry optimizations? If the geometry optimization was done with a different level of theory, it should be included in the model chemistry.